src/main/java/bl4ckscor3/mod/isamodbyvazkii/IsAModByVazkii.java

```diff
@@ -80,7 +90,7 @@ public static void onItemTooltip(ItemTooltipEvent event)
 			{
 				if(TextFormatting.getTextWithoutFormattingCodes(tooltips.get(i)).equals(modName))
 				{
-					tooltips.set(i, I18n.format("isamodbyvazkii:mod", tooltips.get(i)));
+					tooltips.set(i, I18n.format(MOD, tooltips.get(i)));
```

Comment:
What happens when you remove these lines, does the blue mod name still render with "is a mod by Vazkii"? https://github.com/bl4ckscor3/IsAModByVazkii/blob/master/src/main/java/bl4ckscor3/mod/isamodbyvazkii/IsAModByVazkii.java#L79-L86

Reply:
it changes the mod's actual name so it should work

Reply:
Can you try it out and actually remove the lines, please?

Reply:
yes